pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go

```diff
@@ -646,6 +646,11 @@ func (in *PostgresSpec) DeepCopyInto(out *PostgresSpec) {
 			(*in)[i].DeepCopyInto(&(*out)[i])
 		}
 	}
+	if in.SchedulerName != nil {
```

Comment:
Hi, I compiled your PR because I'm interested in using it because I have to use stork as well. When I did `make docker` I'm receiving these errors:
`pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go:649:22: invalid operation: in.SchedulerName != nil (mismatched types string and nil)`
`pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go:651:8: cannot use new(string) (type *string) as type string in assignment`
`pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go:652:3: invalid indirect of *out (type string)`
`pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go:652:11: invalid indirect of *in (type string)`

Reply:
run `./hack/update-codegen.sh` and maybe `make deps` in case you get compiler errors